src/Symfony/Component/Security/Http/Authenticator/AuthenticatorInterface.php

```diff
@@ -23,6 +24,11 @@
  * @author Ryan Weaver <ryan@symfonycasts.com>
  * @author Amaury Leroux de Lens <amaury@lerouxdelens.com>
  * @author Wouter de Jong <wouter@wouterj.nl>
+ *
+ * @method TokenInterface createToken(Passport $passport, string $firewallName) Create a token for the given user.
+ *                                                                              If you don't care about which token class is used or don't really
```

Comment:
I would remove the "or don't really understand what a token is" part.

Reply:
done